### PR TITLE
pa-check po ty

### PR DESCRIPTION
**Pull request by edsnjavier**

### CODE REVIEW

File: schema.txt [hunk 1]
```diff
@@ -1,57 +1,44 @@
...
     adm_sem INT
-}
-
-TABLE program {
-    pgm_id INT [primary key]
-    pgm_name VARCHAR
-    pgm_desc TEXT
-    pgm_units INT
-    PRIMARY KEY
+    adm_pgm_id INT [primary key]
```
Comment (1123Alessandro) [inline schema.txt]: san pala to nakuha?

File: schema.txt [hunk 2]
```diff
@@ -1,57 +1,44 @@
...
-    schlr_type VARCHAR
+TABLE student_enrollment_facts {
+    stud_id INT
+    adm_id INT
+    coll_id INT
+    crs_id INT
+    tui_id INT
+    number_of_courses TEXT
+    tuition_amount_paid TEXT
+    scholarship_amount TEXT
```
Comment (1123Alessandro) [inline schema.txt]: these three derived facts are a bit shaky, sabi kasi the derived facts should only be derived from that same row only based on what I've been searching. 

e.g. sa number of courses kasi we need to query for the entire table for the same person who enrolled in different courses so it's not really a derived fact (?)

maybe we should think of other facts to put here

File: schema.txt (hunk 1 above)
Comment (edsnjavier) [reply schema.txt]: sorry mali yan HAHAHA baka wrong copy paste ko

File: schema.txt (hunk 2 above)
Comment (edsnjavier) [reply schema.txt]: oki oki hahaha nanghula lang rin ako eh, pwede mo naman ibahin yan, tapos wdym pala doon sa same row

Comment (1123Alessandro) [reply schema.txt]: yung derived facts dapat kukunin lang yung info from that one row lang

**Pull request merged**